testing/cli/test_cli_sct_compute_mtsat.py

```diff
@@ -21,14 +26,32 @@
      '-t1', sct_test_path('mt', 't1w.nii.gz'),
      '-omtsat', out_mstat,
      '-ot1map', out_t1map,
-     '-trmt', '51', '-trpd', '52', '-trt1', '10', '-famt', '4', '-fapd', '5', '-fat1', '14'],
+     '-trmt', '0.030', '-trpd', '0.030', '-trt1', '0.015', '-famt', '9', '-fapd', '9', '-fat1', '15'],
```

Comment:
Thanks for making these values consistent with the JSON sidecar.

Reply:
No problem!